08Sep_NumSmallerThanCurrNum.py

```diff
@@ -0,0 +1,45 @@
+#
+# https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/description/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+######################################################################################################################################################
+#
+# Given the array nums, for each nums[i] find out how many numbers in the array are smaller than it. That is, for each nums[i] you have to count the number
+# of valid j's such that j != i and nums[j] < nums[i].
+# Return the answer in an array.
+#
+# Example 1:
+# Input: nums = [8,1,2,2,3]
+# Output: [4,0,1,1,3]
+# Explanation: 
+# For nums[0]=8 there exist four smaller numbers than it (1, 2, 2 and 3). 
+# For nums[1]=1 does not exist any smaller number than it.
+# For nums[2]=2 there exist one smaller number than it (1). 
+# For nums[3]=2 there exist one smaller number than it (1). 
+# For nums[4]=3 there exist three smaller numbers than it (1, 2 and 2).
+#
+# Example 2:
+# Input: nums = [6,5,4,8]
+# Output: [2,1,0,3]
+#
+# Example 3:
+# Input: nums = [7,7,7,7]
+# Output: [0,0,0,0]
+#
+class Solution:
+    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
+
+        res = []
+        # [8, 1, 2, 2, 3]
+        nums_sorted = sorted(nums)
+        # nums_sorted = [1, 2, 2, 3, 8]
+        # index          0  1  2  3  4
+
+        for i in range(len(nums_sorted)):
+            # nums[i] = 8, 1, 2, 2, 3
+            # nums_sorted.index(nums[i]) = [4, 0, 1, 1, 3]
+            res.append(nums_sorted.index(nums[i]))
```

Comment:
I suspect there is a faster way to do this but if it passes LeetCode's checker I guess is fine